aap_auth/backend.py

```diff
@@ -54,7 +53,7 @@ def authenticate(self, request, token=None):
            return None, None
 
        try:
-           decoded_token = verify_token(jwt, AAPAcess().cert)
```

Comment:
Why is it ok to just decode the token rather than verifying it?

Also, the comments on lines 38-52 and 100-101 should be updated if you're removing token verification